### PR TITLE
(FACT-1607) split the default external fact directory into 2 test

### PR DESCRIPTION
**Pull request by ericldelaney**
I split the 2 tests for the default external fact directory into 2 tests
Updated code style, added risk tags, and TestRail numbers

### CODE REVIEW

**Comment by ericldelaney** [comment]
Tested on:
arista4-32a, ciscoxr-64a%7Bhypervisor=vmpooler%7D, cisconx-64a%7Bhypervisor=vmpooler%7D, cumulus25-64a, debian8-64a, fedora25-64a, osx1012-64a, redhat7-64a, sles12-64a, solaris11-64a, ubuntu1604-64a, windows10ent-64a, windows2016-64a

File: acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb [hunk 1]
```diff
@@ -1,95 +1,35 @@
...
+    end
+
+    step "agent #{agent}: resolve the external fact" do
+      on(agent, facter('external_fact')) do |facter_output|
+        assert_equal('external_value', facter_output.stdout.chomp, 'Expected to resolve the external_fact')
+      end
+    end
   end
 end
+
```
Comment (johnduarte) [inline acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb]: This blank line should be removed

**Comment by ericldelaney** [comment]
@johnduarte Updated with your requested changes

File: acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb [hunk 2]
```diff
@@ -1,95 +1,34 @@
...
-  on(agent, "chmod +x '#{ext_fact_3}'")
-
-  step "Agent #{agent}: the fact value from the custom external dir should override that of facts.d"
-  on(agent, facter("--external-dir '#{custom_external_dir}' external_fact_1"))
-  assert_match(/baz/, stdout)
-
-  teardown do
-    on(agent, "rm -f '#{ext_fact_1}' '#{ext_fact_2}' '#{ext_fact_3}'")
+# facter resolves facts from the default facts.d directory
+#      on Unix this can be 3 directories (See fact_directory_precedence.rb)
```
Comment (kurtwall) [inline acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb]: Funky spacing here.

File: acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb [hunk 3]
```diff
@@ -1,95 +1,34 @@
...
+    end
+
+    step "agent #{agent}: resolve the external fact" do
+      on(agent, facter('external_fact')) do |facter_output|
+        assert_equal('external_value', facter_output.stdout.chomp, 'Expected to resolve the external_fact')
+      end
+    end
   end
-end
+end
```
Comment (kurtwall) [inline acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb]: Missing newline?

File: acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb (hunk 2 above)
Comment (ericldelaney) [inline acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb]: I was trying to differentiate between the comment and the paths that come next, I'll remove it.

File: acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb (hunk 3 above)
Comment (ericldelaney) [inline acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb]: Yeah, John was complaining when there too many, so I was removing them, but in RubyMine that means you can end up without 1 at the end of the file if your not careful. I'll fixed it

Comment (kurtwall) [inline acceptance/tests/external_facts/root_uses_default_external_fact_dir.rb]: 😆

**Comment by ericldelaney** [comment]
@kurtwall Updated with your requests

**Comment by kurtwall** [comment]
I'll leave the merge to @johnduarte.

**Pull request merged**